Automattic/vip-go-mu-plugins
# Add option to disable Elasticsearch healthcheck job

## Description

Add option to disable the Elasticsearch healthcheck cronjob by setting a constant.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Example:

1. Check out PR.
1. Sandbox a site with VIP search setup
1. `define( 'DISABLE_VIP_SEARCH_HEALTHCHECKS', true )` in `vip-config.php`
1. Verify health check job is no longer in cron events list
